### ISSUE TITLE
Cannot access the page numbers field

### ISSUE THREAD

**Issue by david-iterators** [CityOfBoston/Iterators#112]
Screengrab/videos (insert file names here)
2020 03 24, Cannot access the page numbers field
[2020 03 24, Cannot access the page numbers field.mp4.zip](https://github.com/CityOfBoston/Iterators/files/4378232/2020.03.24.Cannot.access.the.page.numbers.field.mp4.zip)
.mp4

Link
https://d8-stg.boston.gov/events

Node # (if known)
4

Type of User:  Anonymous

Environment
(Device > OS > Browser)
Computer > MacOS 10.15.3 > Chrome 80

Trusted Tester Test
4

Brief Description
Unable to access the field that advances the user to the next page.mp4

Steps to reproduce
1.  00:00, Go to https://d8-stg.boston.gov/events from the landing page
2.  00:07, Start tabbing through the site using only the tab key.
3.  00:28, Successfully tabbed through the page until trying to tab onto the field used to select a different page. 
	The rest of the video (not seen on the video) is an attempt to tab onto the field so that a new page could be selected or using the Shift Tab to go back and try again.

Desired Result
•	Able to successfully tab through the entire Event page using only the tab button

Actual Result
•	Unable to successfully tab through the entire Event page using only the tab button

**Comment by andrebtlima** [comment]
@subaha-cob you can now access the pagination at the bottom of the page, but you cannot tab to the actual events.

**Comment by subaha-cob** [comment]
Fixed the issue. Going through travis now. Fix should be in by the morning.

**Comment by david-iterators** [comment]
This is mostly fixed on https://d8-uat.boston.gov/events.  There is a functional bug present in that the "..." link can not be used by either a mouse or keyboard.


Screengrab/videos (insert file names here)
2020 06 07, Update to issue 112, Events Page control with Keyboard or Mouse.mp4

Link
https://d8-uat.boston.gov/events

Node # (if known)


Type of User:  Anonymous

Environment
(Device > OS > Browser)
Computer > macOS > Safari

Trusted Tester Test
Keyboard

Brief Description
The “…” field on the page numbers cannot be accessed with either a mouse or keyboard.

Steps to reproduce
1)	00:00, Go to https://d8-uat.boston.gov/events
2)	00:17, Use the keyboard to tab to the page control at the bottom of the list of events.
3)	00:27, BUG:  Note that the “…” link cannot be accessed with the keyboard
a)	BUG:  Separately (not shown on the video), the mouse cannot be used to click on the “…” link.  This also applies to the “…” that would appear at the beginning of the list if the last page link is pressed.


Desired Result
Links accessible through mouse and keyboard

Actual Result
The “…” page control is not active

**Comment by david-iterators** [comment]
[2020 06 07, Update to issue 112, Events Page control with Keyboard or Mouse.zip](https://github.com/CityOfBoston/Iterators/files/4742186/2020.06.07.Update.to.issue.112.Events.Page.control.with.Keyboard.or.Mouse.zip)

**Comment by andrebtlima** [comment]
@subaha-cob the issue came back where you can access the pagination at the bottom of the page, but you cannot tab to the actual events, also the keyboard skips the ... in the pagination
